src/value-converters/date-format-value-converter.ts

```diff
@@ -8,24 +8,50 @@ import moment from "moment";
 export class DateFormatValueConverter {
     private static _defaultFormat = "LLL";
 
+    /**
+     * @deprecated Use instance method {@link setDefaultFormat} instead
+     */
     static setDefaultFormat(format:string) {
         this._defaultFormat = format;
     }
 
+
+    setDefaultFormat(format:string) {
+        DateFormatValueConverter._defaultFormat = format;
```

Comment:
Mmh. I think you should swap the direction in which you delegate: The deprecated function should delegate to the un-deprecated one.
- A new function should not call - esp. nothing but - deprecated code right off the bat.
- The old implementation is simply wrong. A wrapper may hide that, but does not correct it. Calling the new function e.g. in place of a lambda "Java style" would unexpectedly fail only in the static function, only in a way a static function cannot fail. We should not obfuscate the user's error by adding our own.
- It is far easier to drag one's feet on removal of deprecated code when it appears to be complicated. A function that but delegates is "The Ideal Deprecated Artifact", as its removal is obviously save for ourselves and obviously at most a minor inconvenience in user code.

Reply:
`DateFormatValueConvert._defaultFormat` is not deprecated. But methods (instance and static) are doing the same.

Reply:
`DateFormatValueConvert._defaultFormat` is not deprecated. But methods (instance and static) are doing the same.

Reply:
Ah, sorry. I misread this one - I presume because `this._defaultFormat` does not actually exist.
(Since when is this allowed!? I vividly recall wrong member/static scoping to be a compile error in the not-too-distant past - it was one of the first things I noticed when learning TypeScript in 2017.)
I thought I had read the new functions calling the old ones.